src/shared/ptyfwd.c

```diff
@@ -980,6 +980,8 @@ int pty_forward_new(
                         raw_stdin_attr = f->saved_stdin_attr;
                         cfmakeraw(&raw_stdin_attr);
```

Comment:
@poettering Frankly I'm very confused why we need to do the `cfmakeraw()` dance at all. It messes up every other tool writing to the terminal at the same time as the pty-forward process. Since we don't do it at all when `PTY_FORWARD_READ_ONLY` is set, and that works fine, what exactly is the benefit of putting the tty in raw mode?